docs/versions/latest/Release.md

```diff
@@ -39,7 +48,47 @@
 [//]: # (Optional But higlhy recommended Specify *NONE* if missing)
 [//]: # (#### Relevant Documentation:)
 
+### Enabled Unit testing for npm lib based projects
```

Comment:
These release notes are from the previous release. Perhaps an issue with the sync?

Reply:
Hi Stefan. Yeah, this added when I synched the latest changes from main. I have changed it now. Thanks!